lib/qxcli/commands/Compile.js

```diff
@@ -562,28 +374,28 @@ qx.Class.define("qxcli.commands.Compile", {
       });
 
       maker.getAnalyser().addListener("compiledClass", function(evt) {
-        var data = evt.getData();
-        var data = evt.getData();
-        var markers = data.dbClassInfo.markers;
+        let dat = evt.getData();
+        var markers = dat.dbClassInfo.markers;
         if (markers) {
           markers.forEach(function(marker) {
             var str = qxcompiler.Console.decodeMarker(marker);
             t.warn(data.classFile.getClassName() + ": " + str);
```

Comment:
variable is still `data` here

Reply:
Thanks for the hint. In master there is 
`
        var data = evt.getData();
        var data = evt.getData();
`